packages/encoder/test/overload.test.ts

```diff
@@ -279,6 +279,39 @@ describe("Overload resolution", () => {
       );
     });
 
+    it("Prefers bools to strings with strict booleans on", async () => {
+      const { abi, tx } = await encoder.encodeTransaction(
+        "overloaded",
+        ["true"],
+        {
+          allowOptions: true,
+          strictBooleans: true
+        }
+      );
+      assert.lengthOf(abi.inputs, 1);
+      assert.strictEqual(abi.inputs[0].type, "bool");
+      const selector = Codec.AbiData.Utils.abiSelector(abi);
+      assert.strictEqual(
+        tx.data,
+        selector +
+          "0000000000000000000000000000000000000000000000000000000000000001"
+      );
+    });
+
+    it("Encodes as string as last resort with strict booleans on", async () => {
```

Comment:
is the `strictBooleans` option actually on here? I don't see it in the options

Reply:
but that's for the previous test, shouldn't it be on line ~303?

Reply:
....oh wow you're absolutely right.  Oops.  Better fix that, then!